index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <title>Valentin Lopan - Junior Frontend Developer</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <header id="about">
+        <nav>
+            <ul id="nav" class="nav">
+                <li><a class="smoothscroll" href="#about">About</a></li>
+                <li><a class="smoothscroll" href="#skills">Skills</a></li>
+                <li><a class="smoothscroll" href="#works">Works</a></li>
+                <li><a class="smoothscroll" href="#education">Education</a></li>
+            </ul>
+        </nav>
+        <div class="row">
+            <div class="three columns">
+                <img class="profile-pic" src="images/profile.jpg" alt="" />
```

Comment:
It could be better if you put some describe information in alt tag (in whole file)